pilates/beam/preprocessor.py

```diff
@@ -7,7 +7,10 @@
 def copy_plans_from_asim(settings):
     asim_output_data_dir = settings['asim_local_output_folder']
     plans_path = os.path.join(asim_output_data_dir, 'final_plans.csv')
-    beam_local_plans = os.path.join(settings['beam_local_input_folder'], settings['beam_plans'])
+    beam_local_plans = os.path.join(
+    	settings['beam_local_input_folder'],
+    	settings['region'], 'gemini',
```

Comment:
It's better to remove this intermediate 'gemini' folder and put it in the beam_plans setting (as a part of the file path). gemini is a particular scenario within multiple SFBay scenarios.